src/pint/models/tcb_conversion.py

```diff
@@ -0,0 +1,168 @@
+"""TCB to TDB conversion of a timing model."""
+
+import numpy as np
+
+from pint.models.parameter import MJDParameter
+from loguru import logger as log
+
+__all__ = [
+    "IFTE_K",
+    "scale_parameter",
+    "transform_mjd_parameter",
+    "convert_tcb_to_tdb",
+]
+
+# These constants are taken from Irwin & Fukushima 1999.
+# These are the same as the constants used in tempo2 as of 10 Feb 2023.
+IFTE_MJD0 = np.longdouble("43144.0003725")
+IFTE_KM1 = np.longdouble("1.55051979176e-8")
+IFTE_K = 1 + IFTE_KM1
+
+
+def scale_parameter(model, param, n, backwards):
+    """Scale a parameter x by a power of IFTE_K
+        x_tdb = x_tdb * IFTE_K**n
```

Comment:
I'm quite possibly misunderstanding, but should one of the `x_tdb` instead be `x_tcb`?

Reply:
Resolved.